lib/Doctrine/DBAL/Query/Expression/CompositeExpression.php

```diff
@@ -84,6 +88,28 @@ public function add($part)
         return $this;
     }
 
+    /**
+     * Returns a CompositeExpression with the given part added.
+     *
+     * This methods returns a new instance. The current instance is unaffected by this method call.
```

Comment:
IMO, it's too verbose. “Returns a _new_ CompositeExpression with the given part added” should suffice.

Reply:
Updated in d191e953778c24efb0fd4bd3a4752e443563239e.